### PR TITLE
ig: fix %check.

### PR DESCRIPTION
**Pull request by eiffel-fl**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Fix `%check` of `ig`.

###### Change Log  

- Fix `%check`

###### Does this affect the toolchain?  

**NO**

### CODE REVIEW

File: SPECS/ig/ig.spec [hunk 1]
```diff
@@ -52,5 +61,5 @@ rm $ig_file
 
 %changelog
 * Tue Mar 14 2023 Francis Laniel <flaniel@linux.microsoft.com> - 0.25.0-1
```
Comment (PawelWMS) [inline SPECS/ig/ig.spec:63]: One small thing: please bump `Release`. We can no longer skip doing that as we published (or will soon publish) packages to 3.0 preview.

**Review by PawelWMS — APPROVED** [review]

**Review by mfrw — APPROVED** [review]

File: SPECS/ig/ig.spec [hunk 2]
```diff
@@ -34,23 +34,35 @@ mkdir -p "%{buildroot}/%{_bindir}"
...
+# as, for example, the path tested will be inside the chroot while ig reports
+# the full path from host point of view.
+# For all these reasons, we will skip the unit tests and rather run a small
+# integration test.
+# Moreover, Inspektor Gadget CI covers Azure Linux extensively:
+# https://github.com/inspektor-gadget/inspektor-gadget/pull/1186/commits/066bf618d158
 sleep inf &
 sleep_pid=$!
-kill $ig_pid
+./bin/build/ig snapshot process --host | grep -qP "sleep\s+${sleep_pid}"
```
Comment (mfrw) [inline SPECS/ig/ig.spec:54]: Running locally it seems to break:
image

@eiffel-fl I feel this is unexpected.

Comment (mfrw) [reply SPECS/ig/ig.spec:54]: I ran a test build on my build VM

Comment (mfrw) [reply SPECS/ig/ig.spec:54]: Ubuntu 22.x is my build VM.

**Review by mfrw — APPROVED** [review]

**Pull request merged**